packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -362,6 +433,32 @@ function Thread( {
 				commentExcerpt
 		  );
 
+	if ( 'new-note-thread' === thread.id && showCommentBoard && isFloating ) {
+		return (
+			// Disable reason: role="listitem" does in fact support aria-expanded.
+			// eslint-disable-next-line jsx-a11y/role-supports-aria-props
+			<VStack
+				className="editor-collab-sidebar-panel__thread is-selected is-floating"
+				id={ `comment-thread-${ thread.id }` }
+				spacing="3"
+				tabIndex={ 0 }
+				role="listitem"
+				aria-label={ ariaLabel }
+				aria-expanded={ isSelected }
+				ref={ isFloating ? refs.setFloating : undefined }
+				style={ isFloating ? { top: y } : undefined }
+			>
+				<AddComment
+					onSubmit={ onAddReply }
+					showCommentBoard={ showCommentBoard }
+					setShowCommentBoard={ setShowCommentBoard }
+					commentSidebarRef={ commentSidebarRef }
+					reflowComments={ reflowComments }
+				/>
+			</VStack>
```

Comment:
Same as my previous comment about double wrapping. The `aria` attributes probably aren't needed here.